CMSgov/easi-app
# [EASI 3184] Update and  Confirm LCID actions be

# EASI-3184

## Changes and Description

- Mutation to Update an LCID
    - requires that the LCID was previously issued 
- Mutation to Confirm an LCID
    - requires that the LCID was previously issued

The function structure closely follows the code for issuing LCID for consistency.


## How to test this change
Use the Postman collection to do the following
1. Create a new System Intake
2. Issue an LCID
3. Confirm that LCID
4. Update that LCID
5. Confirm that you were able to update the LCIDs, and that an action is recorded for each step
    a. Confirm that the recorded action for confirm and update also populate the previous and new values for an LCID. 


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
